lib/Cake/Test/Case/Utility/FileTest.php

```diff
@@ -62,6 +63,15 @@ public function tearDown() {
  * testBasic method
  *
  * @return void
+ * @covers ::__construct
+ * @covers ::info
+ * @covers ::ext
+ * @covers ::name
+ * @covers ::md5
+ * @covers ::size
+ * @covers ::owner
+ * @covers ::group
+ * @covers ::Folder
```

Comment:
Yup, for them, too.
But see here:
https://phpunit.de/manual/3.7/en/appendixes.annotations.html#appendixes.annotations.coversDefaultClass

> * @coversDefaultClass \Foo\CoveredClass

plus

> * @covers ::publicMethod

results in
> @covers \Foo\CoveredClass::publicMethod

but with way less typing :smirk_cat: 

Doing this for two years now in my test cases.
Works like charm!